### PR TITLE
Update projects.md

### PR DESCRIPTION
**Pull request by xiaou61**
Added my own personal blog

### CODE REVIEW

File: docs/theme/src/zh/demo/projects.md [hunk 1]
```diff
@@ -400,6 +400,14 @@ blog:
     url: https://xxxweii.github.io/XXXWeiiBlog
     repo: https://github.com/XXXWeii/XXXWeiiBlog
     preview: https://xxxweii.github.io/XXXWeiiBlog/assets/images/XXXWeii.png
+
+  - name: 小u的学习笔记
+    desc: 热爱可抵岁月漫长
+    logo: https://mydocs.guoyaxue.top/site_logo.png
+    url: https://mydocs.guoyaxue.top
+    repo: https://github.com/xiaou61/mydocs
+    preview: https://mydocs.guoyaxue.top/site_logo.png
```
Comment (Mister-Hope) [inline docs/theme/src/zh/demo/projects.md:409]: Fix this

**Review by Mister-Hope — CHANGES_REQUESTED** [review]

File: docs/theme/src/zh/demo/projects.md (hunk 1 above)
Comment (xiaou61) [reply docs/theme/src/zh/demo/projects.md:409]: yes

Comment (Mister-Hope) [reply docs/theme/src/zh/demo/projects.md:409]: Fix the preview image

Comment (xiaou61) [reply docs/theme/src/zh/demo/projects.md:409]: I pulled a pull again, the previous one has already been modified,tank you

**Pull request closed without merging**